services/ux-backend/main.go

```diff
@@ -61,7 +60,8 @@ func main() {
 		klog.Info("shutting down!")
 		os.Exit(-1)
 	}
-	http.HandleFunc("/onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
+
+	http.HandleFunc("/onboarding-tokens/{token-type}", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please verify without passing {token-type} to confirm if it's working as expected. I'm asking because generating a token from the UI will not pass this test.

Reply:
Tried a few examples locally and for maintaing backward compatibilty, we have two options,

1. We either manually trim the prefix and match the path
2. Have two endpoints one `/onboarding-tokens/{tokenType}` and the other being `/onboarding-tokens`

Reply:
how about adding query parameters these can be optional.

Reply:
Had a discussion with Ohad offline and decided to keep two different endpoints for client and peer onboarding

Reply:
okay, you can mark this as resolved